.gitignore

```diff
@@ -12,6 +12,7 @@
 *.suo
 *.trs
 *.user
+*.bak
```

Comment:
This was placed next to the ignore for Emacs backup files.

I'll move it up to be in alphabetical order.